lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -371,7 +371,7 @@ def max_thread_limit
     ParallelTests::CLI.new.run(args)
     Rake::Task[:spec_clean].invoke
   rescue LoadError
-    fail 'Add the parallel_tests gem to Gemfile to enable this task'
+    raise 'Add the parallel_tests gem to Gemfile to enable this task'
```

Comment:
what about wrapping the whole task in an `if` statement so it's not even loaded if the gem isn't present? or is a failure better?